### ISSUE TITLE
HD textures

### ISSUE THREAD

**Issue by johnr14** [KozGit/DOOM-3-BFG-VR#204]
Feature request, support HD textures !
There are updates to the texture pack mods on [moddb ](http://www.moddb.com/games/doom-iii): 

There are 2 major mods : 
[UltimateHD v2.1 ](http://www.moddb.com/mods/doom-3-bfg-ultimate/news/doom-3-bfg-ultimatehd-v21)(updated Jul 16 2017)
[Doom 3 BFG Hi def 3.0](http://www.moddb.com/mods/doom-3-bfg-hi-def/news/doom-3-bfg-hi-def-30-released) (updated 3.1d on May 8th, 2017)

Might push the experience closer to Doom4 ?

**Comment by KozGit** [comment]
Just an FYI, the textures themselves ( and even a lot of the models ) from those mods are compatible.  The issue is with the scripts. The mods you listed aren't just higher resolution  textures, they change a lot of gameplay elements via scripting and the entity definitions. Those changes can conflict with some of the changes made in this mod to support VR. I do plan on compiling a hi def pack for VR culled from those items, just haven't gotten to it yet.  

If you want to try, make a backup copy of your d3bfg directory ( with this mod installed ). Look inside the 'base' directory, and make note of all subdirectories inside it. Install one of the hi def packs you listed.  Look inside 'base' again, and you should see a bunch of new directories. Leave all the original directories, but delete all the new directories except for models, materials, and sounds.  Try launching d3bfgvr from the backup with the textures Installed and see what happens( I'm on mobile now , so there's a fair chance I might forgetting something, so only try with a backup!)

**Comment by AerowynX** [comment]
the hd textures work with the mod but honestly they don't make a massive difference overall.. nowhere close to doom 4 or anything haha.. they also can sometimes cause random crashes as well

**Comment by KozGit** [comment]
Yeah, this will never be Doom 2016 :). Some of the textures and item models from the hd pack are nice though, especially when up close.

**Comment by AerowynX** [comment]
yea they make for good screenshots haha but when playing ingame you aren't going to be going OMG it's new DOOM! ha..

**Comment by sycspysycspy** [comment]
I have texture bug (almost everything goes black)  after using "Doom3BFG allinone+ultimateHD"  and remove it and do a fresh install of the game, as i get a lot of unusable saves so I decide to get rid of the hd textures.
